src/Gren/Kernel/HttpServer.js

```diff
@@ -85,6 +85,11 @@ var _HttpServer_setBody = F2(function (body, res) {
   return res;
 });
 
+var _HttpServer_setBodyToBytes = F2(function (bytes, res) {
+  let buffer = Buffer.from(bytes.buffer);
```

Comment:
It's better to use `Uint8Array` here.

For more info: https://sindresorhus.com/blog/goodbye-nodejs-buffer

Reply:
Good to know, thanks!